unitaryhack.md

```diff
@@ -54,8 +54,6 @@ You can claim swag prizes by contributing to these other unitaryHACK issues:
 
 [Add the ISWAP operation](https://github.com/PennyLaneAI/pennylane/issues/1146)
 
-[Allow representing arbitrary operators with the Operator class](https://github.com/PennyLaneAI/pennylane/issues/1120)
```

Comment:
What happened here?

Reply:
Thought of removing it as we've converged towards exploring it more in-depth though saw your PR which looks better. :slightly_smiling_face: